Vagrantfile

```diff
@@ -49,6 +49,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.network "forwarded_port", guest: 9991, host: host_port, host_ip: host_ip_addr
+  config.vm.network "forwarded_port", guest: 9995, host: 9995, host_ip: host_ip_addr
```

Comment:
yup, I missed out one thing regarding nginx proxying which is clear now reading this comment so will fix that.